src/include/ZividPython/Releasable.h

```diff
@@ -2,6 +2,12 @@
 
 #include <optional>
 
+#define ZIVID_PYTHON_FORWARD_0_ARGS_TEMPLATE_WRAP_RETURN(ReturnType, functionName, returnTypetypename)                 \
```

Comment:
returnTypetypename sounds a bit weird. Looks like a type from its usage so it should have uppercase first letter, ReturnTypeTypename would be better.

I also se that there is already some inconsistency in this file. arg1type below is a type as well, ReturnType is also a type. The cleanest is probably to use camelCase for all macro args everything, and not CapiTaliZed at all.

An other thing is that I have a hard time understanding how ReturnTypeTypename is different from ReturnType. Maybe call it TempleteArg1 or templateArg1.

For concistency the macro should be called ZIVID_PYTHON_FORWARD_0_ARGS_TEMPLATE_1_ARG_WRAP_RETURN

Reply:
Ok, will change the name of the macro, but I will do the other cleanup in a separate PR: https://github.com/zivid/zivid-python/pull/32